src/generate/cs_type.rs

```diff
@@ -240,6 +240,8 @@ pub trait CSType: Sized {
                     data: "".to_string(),
                     comment: Some("Methods".to_string()),
                 }));
+
+            cpp_type.declarations.reserve(5 * t.method_count as usize);
```

Comment:
Why 5? If it's to improve perf by allocating early, provide a comment here so we aren't left wondering in the future